diff --git a/node_modules/redux/es/redux.js b/node_modules/redux/es/redux.js
index 9236d7a..bf9bbde 100644
--- a/node_modules/redux/es/redux.js
+++ b/node_modules/redux/es/redux.js
@@ -639,7 +639,7 @@ function applyMiddleware() {
         }
       };
       var chain = middlewares.map(function (middleware) {
-        return middleware(middlewareAPI);
+        return (middleware.default || middleware)(middlewareAPI);
       });
       _dispatch = compose.apply(void 0, chain)(store.dispatch);
       return _objectSpread2({}, store, {
